pkg/support/dump.go

```diff
@@ -293,7 +295,13 @@ func (d *agentDumper) DumpAgentInfo(basedir string) error {
 }
 
 func (d *agentDumper) DumpMemberlist(basedir string) error {
-	return dumpAntctlGet(d.fs, d.executor, "memberlist", basedir)
+	var ml []memberlist.Response
```

Comment:
The real issue is memberlistCluster is nil on windows. The change in the PR can't resolve the problem.

The problem also exists on Linux when Egress feature is disabled. `memberlist.HandleFunc` should check whether `GetMemberlistCluster()` return nil before calling its `AliveNodes` method. If the interface is nil, it should return a proper http code and message to client indicating memberlist is not running.